### ISSUE TITLE
SS 3.5 Viewer Groups not loading, unable to select

### ISSUE THREAD

**Issue by RVXD** [silverstripe/silverstripe-secureassets#55]
With latest 3.5 version it seems impossible to select Viewer Groups on Folders.

Retrieving the tree fails: http://test.dev/admin/assets/EditForm/field/ViewerGroups/tree

Response: I can't handle sub-URLs on class CMSForm.

![screen shot 2017-03-16 at 10 33 03](https://cloud.githubusercontent.com/assets/1586761/23989893/8c28c8a6-0a34-11e7-8643-9a5b0e28703f.png)

**Comment by rasstislav** [comment]
Select Viewer Groups not works on SS 3.4 and 3.5 ...

**Comment by robbieaverill** [comment]
Thanks @RVXD @rasstislav - could you provide the version of secure assets you're using?

**Comment by RVXD** [comment]
I'm still having the issue with this newer combination:
"name": "silverstripe/framework", "version": "3.6.0",
"name": "silverstripe/secureassets","version": "1.1.4",

**Comment by rasstislav** [comment]
"silverstripe/secureassets": "1.1.4"

**Comment by dhensby** [comment]
I've installed a vanilla 3.6.0 with secureassets 1.1.4 and can't replicate this issue.

I've also had no problems with 3.x-dev of silverstripe and 1.1.x-dev of secure assets.

composer.json [require section]:

```json
"require": {
                "php": ">=5.3.3",
                "silverstripe/cms": "3.6.0",
                "silverstripe/framework": "3.6.0",
                "silverstripe/reports": "3.6.0",
                "silverstripe/siteconfig": "3.6.0",
                "silverstripe-themes/simple": "3.1.*",
                "silverstripe/secureassets": "1.1.4"
        },
```

**Comment by MilesSummers** [comment]
I have just tried a vanilla SS3.6.1 install with secureassets dev-master and have the same issue as RVXD

**Comment by silverstripe-elliot** [comment]
@dhensby I'm having the same issue with a vanilla version of 3.6.1 installed. @sheadawson's pull request (https://github.com/silverstripe/silverstripe-secureassets/pull/56) fixes this by using Listbox instead of TreeDropdown

**Comment by robbieaverill** [comment]
I can't reproduce this either on SS 3.6.1 and secure assets 1.1.4.

@MilesSummers @silverstripe-elliot @hpeide @rasstislav @RVXD are you using extra module on top of core? For example, the screenshot looks like RVXD is using fluent. Perhaps there's another module interfering with this somehow, but nor @dhensby or I can reproduce this bug.

**Comment by MilesSummers** [comment]
No I did a fresh install of SS 3.6.1 and secure assets to eliminate any interference from other modules

**Comment by robbieaverill** [comment]
Another fresh installation, cannot reproduce still. 

![image](https://user-images.githubusercontent.com/5170590/28948875-e63f5606-790c-11e7-9e18-602e1c0689bb.png)

Composer:

```
silverstripe-themes/simple   3.1.0   The SilverStripe simple theme (default SilverStripe 3 theme)
silverstripe/cms             3.6.1   The SilverStripe Content Management System
silverstripe/framework       3.6.1   The SilverStripe framework
silverstripe/reports         3.6.1   Reports module for SilverStripe CMS
silverstripe/secureassets    1.1.4   Adds access restrictions to folders in assets, similar to the way you secure...
silverstripe/siteconfig      3.6.1   Site wide settings administration.
```

Environment: OSX with PHP 5.6 (with xdebug), local Apache 2.4 and MySQL 5.7

---

I do note that the viewer groups dropdown should not be visible until you've selected the "only these groups" option from the radio group, however it's displayed by default on page load. Different issue though.

**Comment by MilesSummers** [comment]
![screenshot_20170804_102945](https://user-images.githubusercontent.com/4961437/28949527-976752ba-7900-11e7-95d8-5f15abe0f4fe.png)

Environment: archlinux with PHP 5.6.30, local Apache 2.4.26 and MariaDB 10.1.25. Chrome browser.

**Comment by robbieaverill** [comment]
Does #56 fix it for you?

**Comment by MilesSummers** [comment]
#56 fixes the problem

**Comment by silverstripe-elliot** [comment]
Same as the others, I was using a fresh installation of 3.6.1 with only secureassets installed.

**Comment by robbieaverill** [comment]
Ok well if #56 fixes it for you and doesn't break it for me then we should merge it :-D

**Comment by robbieaverill** [comment]
Fix available in 1.1.5
